src/util/getMigratedMapConfig.js

```diff
@@ -79,15 +79,27 @@ const upgradeGisAppLayers = (config) => {
 }
 
 // Change layer name from boundary to orgUnit when loading an old map
+// Change colorScale from string to array
 // TODO: Change in db with an upgrade script
-const renameBoundaryLayerToOrgUnitLayer = (config) => ({
-    ...config,
-    mapViews: config.mapViews.map((v) =>
-        v.layer === 'boundary'
-            ? {
-                  ...v,
-                  layer: 'orgUnit',
-              }
-            : v
-    ),
-})
+const upgradeMapViews = (config) => {
+    if (
+        config.mapViews.find(
+            (view) =>
+                view.layer === 'boundary' || typeof view.colorScale === 'string'
+        )
+    ) {
+        return {
+            ...config,
+            mapViews: config.mapViews.map((view) => ({
+                ...view,
+                layer: view.layer === 'boundary' ? 'orgUnit' : view.layer,
+                colorScale:
```

Comment:
Could you add a jest test for this?

Reply:
Jest test added for colorScale translation, we already had a test for changing layer name.